docs/source/tutorials/object_detection.ipynb

```diff
@@ -119,10 +119,21 @@
    "outputs": [],
    "source": [
     "import pickle\n",
-    "\n",
-    "from cleanlab.object_detection.rank import get_label_quality_scores, issues_from_scores\n",
+    "import numpy as np\n",
+    "from cleanlab.internal.object_detection_utils import calculate_bounding_box_areas\n",
     "from cleanlab.object_detection.filter import find_label_issues\n",
-    "from cleanlab.object_detection.summary import visualize"
+    "from cleanlab.object_detection.rank import (\n",
+    "    _separate_label,\n",
+    "    _separate_prediction,\n",
+    "    get_label_quality_scores,\n",
+    "    issues_from_scores,\n",
+    ")\n",
+    "from cleanlab.object_detection.summary import (\n",
+    "    bounding_box_size_distribution,\n",
+    "    class_label_distribution,\n",
+    "    object_counts_per_image,\n",
+    "    visualize,\n",
```

Comment:
```suggestion
```

Reply:
let's also save all these imports for later in the Exploratory Data Analysis section